osbs/conf.py

```diff
@@ -286,6 +286,10 @@ def get_source_registry_uri(self):
     def get_pulp_registry(self):
         return self._get_value("pulp_registry_name", self.conf_section, "pulp_registry_name")
 
+    def get_pulp_manifest_lists(self):
+        return self._get_value("pulp_manifest_lists", self.conf_section,
```

Comment:
Is `pulp_manifest_lists` the correct name here? 
One could run atomic-reactor without pulp and this setting is still relevant.
Maybe something like `use_manifest_list` provides a clearer intent?